### PR TITLE
improve about page

### PR DESCRIPTION
**Pull request by tmsagarofficial**
This is a PR to address the issue as discussed in #76

### CODE REVIEW

File: site/style/about-styles.css [hunk 1]
```diff
@@ -1,32 +1,86 @@
-.sub-topic{
-   font-size: x-large;
+/* Import Google Fonts */
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@300;400;600;700&family=Roboto+Slab:wght@400;700&display=swap');
+
+body {
+    font-family: 'Poppins', sans-serif;
```
Comment (Abhilekhgautam) [inline site/style/about-styles.css:5]: Applying this font family to the entire body brings inconsistencies in nav links with other pages. Consider fixing this.

**Comment by Abhilekhgautam** [comment]
Nice work @tmsagarofficial ! But there is a issue with your submission:

The font in the about page for the nav links is not consistent with other pages:

image

File: site/style/about-styles.css (hunk 1 above)
Comment (tmsagarofficial) [reply site/style/about-styles.css:5]: So what do u think should be done here?

We don't interfere withe the nav link text font?

Comment (Abhilekhgautam) [reply site/style/about-styles.css:5]: @tmsagarofficial  I think applying the font-family to the main class only should fix this.

**Comment by Abhilekhgautam** [comment]
@tmsagarofficial there is still a inconsistency with the size of active tab on the nav links:

Look at the padding at the bottom of the active tab.

image
 
But this is what you have:

image

**Comment by tmsagarofficial** [comment]
Ohhhh okk, I'll check this

**Comment by Abhilekhgautam** [comment]
@tmsagarofficial there is some problem with responsiveness of the sub topics

image

The underlines and text are overlapped.

**Comment by tmsagarofficial** [comment]
Ohh let me look into that as well

**Comment by Abhilekhgautam** [comment]
great work @tmsagarofficial .

Thank You again.

**Pull request merged**